remote-curl.c

```diff
@@ -1046,6 +1046,9 @@ static int fetch_git(struct discovery *heads,
 	struct argv_array args = ARGV_ARRAY_INIT;
 	struct strbuf rpc_result = STRBUF_INIT;
 
+	if (core_use_gvfs_helper)
```

Comment:
I'm concerned this will effectively break `remote-curl` aka `remote-https` any time that `gvfs-helper` is configured (and without regard for whether it is in use by the current process ancestry.

Reply:
As far as I know, when using `gvfs-helper` we should avoid `fetch-pack` at all times. We _never_ want to ask for a pack in the traditional way. Am I wrong?

The only thing I can think of is if we are fetching a submodule, but we do not support that case right now.